Makefile

```diff
@@ -330,12 +328,34 @@ ifneq ($(wildcard $(XSMM_DIR)/lib/libxsmm.*),)
 endif
 
 # OCCA Backends
+OCCA_BACKENDS = /cpu/occa/serial
 ifneq ($(wildcard $(OCCA_DIR)/lib/libocca.*),)
+  OCCA_OMP_MODE_FOUND := $(shell $(OCCA_DIR)/bin/occa modes | grep -q "OpenMP" && echo 1)
```

Comment:
Perhaps avoid shelling out so much and do the text processing in make instead of pipelines?
```makefile
OCCA_MODES := $(shell $(OCCA_DIR)/bin/occa modes)
OCCA_BACKENDS += $(if $(filter OpenMP,$(OCCA_MODES)),/cpu/occa/openmp)
```

Reply:
Agreed and updated